### ISSUE TITLE
Brain Tumor Detection

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#77]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Brain Tumor Detection</br>
:red_circle: **Aim** : Create a DL model which will detect the brain tumors.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/ahmedhamada0/brain-tumor-detection</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by SunitTrivedi** [comment]
Hey, @abhisheks008 I would like to work on this issue.

Full name : Sunit Trivedi
GitHub Profile Link : https://github.com/SunitTrivedi
Email ID : trivedi.sunit@mitaoe.ac.in / sunittrivedi23@gmail.com
Approach for this Project : Using Convolutional Neural Network and and Pytorch Deep Learning Framework.
Participant role : SSOC 2022

**Comment by LahariBoni** [comment]
Assigned it to you @SunitTrivedi. Go Ahead.

**Comment by abhisheks008** [comment]
@SunitTrivedi updates please.

**Comment by abhisheks008** [comment]
Updates please @SunitTrivedi

**Comment by Satyawan7** [comment]
Hey @abhisheks008 I would like to work on this issue.
If no one is working actively on the issue please consider me for this issue.
Full Name: Satyawan Bhuyan
GitHub Profile Link : https://github.com/Satyawan7
Email Id: bhuyansaku19it@student.mes.ac.in
Approach for this Project : Using Convolutional Neural Network
Participant role : SSOC 2022

**Comment by abhisheks008** [comment]
Issue has been unassigned from @SunitTrivedi due to inactivity. Issue has been assigned to @Satyawan7

**Comment by abhisheks008** [comment]
@Satyawan7 updates please.

**Comment by Gaurav762** [comment]
Hey @abhisheks008 I would like to work on this issue.
If no one is working actively on the issue please consider me for this issue.
Full Name: Gaurav Bhoir
GitHub Profile Link : https://github.com/Gaurav762
Email Id: [bhoirgava19it@student.mes.ac.in](mailto:bhoirgava19it@student.mes.ac.in)
Approach for this Project : Using Convolutional Neural Network
Participant role : SSOC 2022

**Comment by Satyawan7** [comment]
Yes we are working on it @abhisheks008

**Comment by Gaurav762** [comment]
Hey @abhisheks008 Let me work on this project if you have not assigned anyone for this project.
Full Name: Gaurav Bhoir
GitHub Profile Link : https://github.com/Gaurav762
Email Id: bhoirgava19it@student.mes.ac.in
Approach for this Project : Using Convolutional Neural Network
Participant role : SSOC 2022

**Comment by abhisheks008** [comment]
@Satyawan7 the program has been ended up. Hence the issue is unasigned from you. Do you wanna continue with this issue as a contributor, as this contribution will not be a part of the Social Summer of Code.

**Comment by Satyawan7** [comment]
Yes I will continue as a contributor

**Comment by abhisheks008** [comment]
Thanks @Satyawan7 for the response. Issue assigned to you @Satyawan7.

@Gaurav762 you can check out other open issues present here in this repo.

**Comment by Abhilash1781** [comment]
Hi , I want to work on this issue . Please assign it to me .

**Comment by Abhilash1781** [comment]
@abhisheks008 I would like to work on this issue . 
Full Name - Abhilash S Bharadwaj
Github Profile link -  https://github.com/Abhilash1781
Email ID - abhilashjambur@gmail.com
Role - SWOC 3.0 Participant 
Approach - I will try using VGG - 16 model

**Comment by abhisheks008** [comment]
Coding period hasn't started yet for DL-Simplified. Keep updating yourself in the discord channel, will notify about the starting.

**Comment by adityagoutam23** [comment]
Hey, @abhisheks008 I would love to work on this issue,
Please find all the details, mentioned for the issue to be assigned.

Full name : Aditya Goutam
GitHub Profile Link : https://github.com/adityagoutam23
Email ID : adityagoutam2318@gmail.com
Approaches for this Project :- Using the image dataset available to predict whether the brain image has tumor or not. Firstly converting the dataset into binary kind of data then using it to train a machine learning algorithm for prediction.
What is your participant role: contributor at SWOC 3.0

**Comment by Abhilash1781** [comment]
The coding period is started , can I work on this issue @abhisheks008

**Comment by abhisheks008** [comment]
I like your approach @Abhilash1781 . Try to implement various deep learning algos atleast 3, then compare their accuracies to find out the best fitted one.

Issue assigned to you @Abhilash1781 

****************************************

@adityagoutam23  issues are being assigned in FCFS basis. You can check out other issues, there are more than 50 issues in this repository. Hope you get it!
